lib/rucio/common/exception.py

```diff
@@ -1139,3 +1139,13 @@ def __init__(self, package: str, *args):
         super(PolicyPackageIsNotVersioned, self).__init__(package, *args)
         self._message = 'Policy package %s does not include information about which Rucio versions it supports.' % self.package
         self.error_code = 109
+
+
+class UnsupportedMetadataPlugin(RucioException):
+    """
+    RucioException
```

Comment:
Could you expand this docstring with more specific information to this exception?

Reply:
yep, added